consul/src/main/scala/io/buoyant/consul/v1/ConsulApi.scala

```diff
@@ -185,5 +189,7 @@ case class ServiceNode(
   ServiceTags: Option[Seq[String]],
   ServiceAddress: Option[String],
   ServicePort: Option[Int],
-  Status: Option[HealthStatus.Value]
+  Status: Option[HealthStatus.Value],
+  ServiceMeta: Option[MetaData],
```

Comment:
I think metadata is usually written as one word which means it would be capitalized as `Metadata`.